### PR TITLE
Changes for azure

### PR DESCRIPTION
**Pull request by hcherukuri**

### CODE REVIEW

File: group_vars/database.yml [hunk 1]
```diff
@@ -11,6 +11,6 @@ mariadb_databases:
   - name: keycloak
 mariadb_mysql_users:
   - name: keycloak-user
-    hosts: "{{ (groups['rhsso'] + groups['datagrid']) | flatten | map('regex_replace', '$', '%') | list }}"
+    hosts: "{{ (groups['rhsso'] + groups['datagrid']) | flatten | map('regex_replace', '_[a-z0-9]*$', '%') | list }}"
```
Comment (guidograzioli) [inline group_vars/database.yml:14]: do we still require the changed regex?

File: group_vars/datagrid.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ jdg_db_pass: keycloak-pass
 
 jdg_default_realm_tls: True
 jdg_keystore_path: /etc/pki/java/keystore.jks
-jdg_keystore_alias: "*.rhssocrossdc.com"
+jdg_keystore_alias: "*.internal.cloudapp.net"
```
Comment (guidograzioli) [inline group_vars/datagrid.yml]: pls reuse the domain_name variable defined in group_vars/all.yml here

File: group_vars/database.yml (hunk 1 above)
Comment (hcherukuri) [reply group_vars/database.yml:14]: We have fixed in the inventory file. I think it won't be necessary.

File: group_vars/datagrid.yml (hunk 2 above)
Comment (hcherukuri) [reply group_vars/datagrid.yml]: Changes added, thank you for pointing out.

**Review by guidograzioli — APPROVED** [review]
LGTM

**Pull request merged**